src/duqtools/create.py

```diff
@@ -211,7 +211,8 @@ def create_run(self, model: Run, *, force: bool = False):
 
         self.system.update_imas_locations(run=model.dirname,
                                           inp=model.data_in,
-                                          out=model.data_out)
+                                          out=model.data_out,
+                                          cfg_filename=self.template_drc.name)
```

Comment:
Consider renaming to `template` or `template_path`

```suggestion
                                          cfg_filename=self.template_path.name)
```